app/src/main/java/com/example/drawingApp/MainActivity.kt

```diff
@@ -12,6 +14,8 @@ import com.example.drawingApp.CustomViews.DrawingFieldView
 import com.example.drawingApp.DataClasses.Hsv
 import com.example.drawingApp.Utils.ColorPickerUtility
 import com.example.drawingApp.Utils.DialogUtility
+import com.example.drawingApp.Utils.ImageUtility.getBitmapFromUriString
```

Comment:
Don't import static methods like this, just call it statically.

Reply:
fair enough, I believe I should change the import to just stop at the ImageUtility and then I can call them staticaly.